challenge_14/action_controller.rb

```diff
@@ -23,6 +24,44 @@ class Blog < ActiveRecord::Base; end
   end
 end
 
+class AppController < ActionController::Base
+  def root
+    response = ""
+    response += "<p><strong>Submit a new Blog Post!</p></strong>"
+    response += "<form method='post' enctype='application/x-www-form-urlencoded' action='/create-post'>"
+    response += "<p><label>Blog Title: <input name='title'></label></p>"
+    response += "<p><label>Content: <textarea name='content'></textarea></label></p>"
+    response += "<p><button>Submit post</button></p>"
+    response += "</form>"
+    render html: response.html_safe
```

Comment:
I realise now that I misspoke when we were pairing: while `#html_safe` is indeed monkey-patched into `String`, it actually [returns an instance](https://github.com/rails/rails/blob/v6.1.3.1/activesupport/lib/active_support/core_ext/string/output_safety.rb#L312-L314) of `ActiveSupport::SafeBuffer`, not another `String` object as I claimed. (`SafeBuffer` is a subclass of `String` so this is only a small difference, but I didn’t want to mislead you!)

Reply:
Thank you for following up on this! ❤️